### ISSUE TITLE
 Links must have discernible text (.k-grid-filter[href="#"])

### ISSUE THREAD

**Issue by sryeluri** [telerik/kendo-angular#2040]
Below is an accessibility issue that needs investigation.

Issue Details
Links must have discernible text

Snippet
<a class="k-grid-filter" href="#" tabindex="-1"><span class="k-icon k-filter"></span></a>

Expected Fix : 
- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="presentation"
- Element's default semantics were not overridden with role="none"

Environment
Chrome version 70.0.3538.77

**Comment by ipeshev** [comment]
@sryeluri thank you for the bug report we will investigate it further.

**Comment by tsvetomir** [comment]
Closing as duplicate of https://github.com/telerik/kendo-angular/issues/1710

**Comment by kraay** [comment]
@tsvetomir Created issue [4860 ](https://github.com/telerik/kendo-ui-core/issues/4860)in kendo-ui-core as our application uses AngularJS.
